pkg/ccl/backupccl/restore_data_processor.go

```diff
@@ -160,6 +163,8 @@ func (rd *restoreDataProcessor) Start(ctx context.Context) {
 	rd.input.Start(ctx)
 
 	ctx, cancel := context.WithCancel(ctx)
+	ctx, rd.agg = bulkutil.MakeTracingAggregatorWithSpan(ctx, fmt.Sprintf("%s-aggregator", restoreDataProcName), rd.EvalCtx.Tracer)
```

Comment:
If we have an aggregator per restore worker as discussed below, I guess this span gives us a view of all the ingestion occurring across our workers? It could be useful, I think it'd help if we ran a longish restore and actually saw these tags in action. Can you maybe run a quick restore an see this playout on `tracez`?

Reply:
Yes, it does give an overall view of the ingestion among all workers